src/test/unit/cli_unittest.cc

```diff
@@ -0,0 +1,306 @@
+        .pgn = PG_RESERVED_FOR_TESTING_1,
+        .offset = 0
+    };
+
+    printf("\n===============================\n");
+    int8_t *data = (int8_t *)getValuePointer(&cval);
+    for(int i=0; i<3; i++){
+        printf("data[%d] = %d\n", i, data[i]);
+    }
+    printf("\n===============================\n");
+
+
+    EXPECT_EQ(123, data[0]);
+    EXPECT_EQ( -3, data[1]);
+    EXPECT_EQ(  1, data[2]);
+
+
+    //cliGet((char *)"osd_item_vbat");
+    //EXPECT_EQ(false, false);
+}
+
+// STUBS
+extern "C" {
+
+float motor_disarmed[MAX_SUPPORTED_MOTORS];
+
+uint16_t batteryWarningVoltage;
+uint8_t useHottAlarmSoundPeriod (void) { return 0; }
+const uint32_t baudRates[] = {0, 9600, 19200, 38400, 57600, 115200, 230400, 250000, 400000}; // see baudRate_e
+
+uint32_t micros(void) {return 0;}
+
+int32_t getAmperage(void) {
+    return 100;
+}
+
+uint16_t getBatteryVoltage(void) {
+    return 42;
+}
+
+batteryState_e getBatteryState(void) {
+    return BATTERY_OK;
+}
+
+uint8_t calculateBatteryPercentageRemaining(void) {
+    return 67;
+}
+
+uint8_t getMotorCount() {
+    return 4;
+}
+
+
+void setPrintfSerialPort(struct serialPort_s *serialPort) { UNUSED(serialPort); }
+
+void tfp_printf(const char * expectedFormat, ...) {
+    va_list args;
+
+    va_start(args, expectedFormat);
+    vprintf(expectedFormat, args);
+    va_end(args);
+}
+
+
+void tfp_format(void *putp, void (*putf) (void *, char), const char * expectedFormat, va_list va) {
+    UNUSED(putp);
+    UNUSED(putf);
+    vprintf(expectedFormat, va);
+}
+
+
+
+static const box_t boxes[] = {
+    { 0, "DUMMYBOX", 0 }
+};
+
+const box_t *findBoxByPermanentId(uint8_t permanentId) {
+	UNUSED(permanentId);
+	return &boxes[0];
+}
+const box_t *findBoxByBoxId(boxId_e boxId) {
+	UNUSED(boxId);
+        return &boxes[0];
+}
+
+const pidConfig_t pgResetTemplate_pidConfig = { 0 };
+const systemConfig_t pgResetTemplate_systemConfig = { .debug_mode = 0,0};
+const mixerConfig_t pgResetTemplate_mixerConfig = { 0, false }; 
+const featureConfig_t pgResetTemplate_featureConfig = { 0 };
+
+int8_t unitTestDataArray[3];
+
+void pgResetFn_unitTestData(int8_t *ptr) {
+    ptr = &unitTestDataArray[0];
+}
+
+void pgResetFn_osdConfig(osdConfig_t *config) { UNUSED(config); }
+void pgResetFn_rxConfig(rxConfig_t *config) { UNUSED(config); }
+void pgResetFn_serialConfig(serialConfig_t *config) { UNUSED(config); }
+void pgResetFn_rxChannelRangeConfigs(rxChannelRangeConfig_t *config) { UNUSED(config); }
+void pgResetFn_rxFailsafeChannelConfigs(rxFailsafeChannelConfig_t *config) { UNUSED(config); }
+
+
+void pgResetFn_ledStripConfig(ledStripConfig_t *ledStripConfig) { UNUSED(ledStripConfig); }
+void pgResetFn_mixerConfig(mixerConfig_t *mixerConfig) { UNUSED(mixerConfig); }
+void pgResetFn_servoParams(servoParam_t *servoParams) { UNUSED(servoParams); }
+
+
+uint32_t getBeeperOffMask(void) { return 0; }
+uint32_t getPreferredBeeperOffMask(void) { return 0; }
+
+void beeper(beeperMode_e mode) {UNUSED(mode);}
```

Comment:
Generally style for stub functions is (eg):
```
void beeper(beeperMode_e) {}
```
By omitting the variable name you avoid the need for `UNUSED`.